src/Object/Datasource/Datasource.php

```diff
@@ -156,15 +174,17 @@ public function setProperties($properties)
      * @return Datasource
      * @throws KlipfolioApiException
      */
-    public function addProperties($properties)
+    public
+    function addProperties($properties)
```

Comment:
Guys, my IDE did this for some reason, I'll fix it. You really don't have to comment on every single instance of this occuring.